### ISSUE TITLE
Cavy run-android

### ISSUE THREAD

**Issue by CasperBartholomaus** [pixielabs/cavy-cli#3]
With the "Cavy run-android" command i get this error:

```
cavy: running `react-native run-android`...
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: spawn react-native ENOENT
    at _errnoException (util.js:1022:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
    at onErrorNT (internal/child_process.js:372:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```

**Comment by jalada** [comment]
Hey @CasperBartholomaus apologies for not getting back to you. We've been really busy at Pixie Labs the last few weeks. We should have some time to look into this for you soon :)

**Comment by em-cd** [comment]
Hi @CasperBartholomaus - that looks like a problem with react-native, not cavy-cli. Are you able to run `react-native run-android` without error?

**Comment by CasperBartholomaus** [comment]
Hi @em-cd , I can run  `react-native run-android` without any problem.

**Comment by sam-vdp** [comment]
I get the same error in a Windows environment. As a workaround, I modified cavy-cli/cavy.js:
`let rn = spawn('react-native', [reactNativeCommand], {stdio: 'inherit', shell: true});`
(The `shell: true` parameter seems to fix the issue)

**Comment by AbigailMcP** [comment]
@CasperBartholomaus - sorry it's been a while! Just checking - were you running cavy-cli in a windows environment too?

**Comment by jalada** [comment]
I'm going to close this as we've not heard anything back, but please comment if you have more information for us :)

**Comment by wilcoxky** [comment]
I am running this on windows and the workaround @sam-vdp suggested fixes the error.

**Comment by mikasius** [comment]
> I am running this on windows and the workaround @sam-vdp suggested fixes the error.

@wilcoxky, can you help? Where you put this suggested fix by @sam-vdp?

**Comment by mikasius** [comment]
In my case found another place: `./cavy-cli/src/runTests.js`.
Added arg: `shell: true`.

**Environment**: Windows
**Cavy**: v3.0.0

Before:
```JS
let rn = spawn("react-native", [command, ...args], { stdio: "inherit" });
```

After: 
```JS
let rn = spawn("react-native", [command, ...args], {
    stdio: "inherit",
    shell: true
  });
```

**Comment by AbigailMcP** [comment]
Hey guys - thanks for resurrecting this 🧟 

I'm gonna test this solution and hopefully roll out a patch for y'all.

**Comment by AbigailMcP** [comment]
☝️ Fix for windows in v1.3.2.
